### ISSUE TITLE
[4.0] login page heading structure

### ISSUE THREAD

**Issue by unknown-sudo** [joomla/joomla-cms#26313]
Hello, I have seen the video from Joomladay Germany about Joomla 4 and would like to help. 
On the login page of Joomla 4 there are a few small problems regarding accessibility which I would like to report divided into several issues. 
 ### What needs to be fixed
The headings structure is wrong and must be fixed. There is an h1 heading and an h3 heading. Either an h2 heading is missing or the h3 heading must be converted to an h2.

### Why this should be fixed
For accessibility it is important that headings are used in order.

### How would you fix it
Either add an h2 or change the h3 to h2.

### Side Effects expected
The CSS may need to be adjusted to avoid the headline being too large.

**Comment by HLeithner** [comment]
Off-topic:
Nice to hear that my talk found some attention. Thx for coming here.

**Comment by unknown-sudo** [comment]
Thank you I hope that my experience here can be brought in well.

**Comment by brianteeman** [comment]
Thanks for reporting this. We fixed this several times before and the template team kept breaking it

**Comment by Quy** [comment]
Please test PR #26394<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26313">issues.joomla.org/tracker/joomla-cms/26313</a>.</sub>
